Compilation issue

Using version `1.2.0` with `@chakra-ui/react` version `1.0.1`, getting this compilation error:

```
wait  - compiling...
error - ./node_modules/chakra-ui-autocomplete/dist/chakra-ui-autocomplete.cjs.development.js:15:0
Module not found: Can't resolve '@chakra-ui/core'
null
```

I think this library needs to be updated for the latest Chakra release or I need to add `@chakra-ui/core` as a dep?

--- a/README.md
+++ b/README.md
@@ -122,7 +122,7 @@ export default function App() {
 
 ```jsx
 import React from 'react'
-import { Text, Flex, Avatar } from '@chakra-ui/core'
+import { Text, Flex, Avatar } from '@chakra-ui/react'
 import { CUIAutoComplete } from 'chakra-ui-autocomplete'
 
 

--- a/example/package.json
+++ b/example/package.json
@@ -10,7 +10,7 @@
 		"eject": "node ../node_modules/react-scripts/bin/react-scripts.js eject"
 	},
 	"dependencies": {
-		"@chakra-ui/core": "^1.0.0-rc.8",
+		"@chakra-ui/react": "^1.0.1",
 		"framer-motion": "^2.9.4",
 		"@testing-library/jest-dom": "file:../node_modules/@testing-library/jest-dom",
 		"@testing-library/react": "file:../node_modules/@testing-library/react",

--- a/example/src/App.tsx
+++ b/example/src/App.tsx
@@ -1,5 +1,5 @@
 import React from 'react'
-import { ThemeProvider, CSSReset, Text, Flex, Heading, Avatar } from '@chakra-ui/core'
+import { ThemeProvider, CSSReset, Text, Flex, Heading, Avatar } from '@chakra-ui/react'
 import { CUIAutoComplete } from '../../dist'
 import theme from './theme'
 

--- a/example/src/theme.tsx
+++ b/example/src/theme.tsx
@@ -1,7 +1,20 @@
-import { theme as chakraTheme } from '@chakra-ui/core';
+// import { theme as chakraTheme } from '@chakra-ui/react';
+import { extendTheme } from '@chakra-ui/react'
+import { createBreakpoints } from '@chakra-ui/theme-tools'
+
+// const fonts = { mono: `'Menlo', monospace`, }
+
+// const theme = extendTheme({
+//   colors: {
+//     black: '#16161D',
+//   },
+//   fonts,
+//   breakpoints,
+// })
+
+// export default theme
 
 const fonts = {
-	...chakraTheme.fonts,
 	mono: `'Menlo', Monaco, Fira Code, Ubuntu Mono, monospace`,
 	heading: `"Graphik Web", 'Inter', 'Ubuntu', Cantarell, Oxygen, sans-serif`,
 	body: `"Graphik Web", 'Inter', 'Ubuntu', Segoe UI, Cantarell, Oxygen, Roboto, Fira Sans, Helvetica Neue, sans-serif`
@@ -20,7 +33,6 @@ const fontSizes = {
 };
 
 const colors = {
-	...chakraTheme.colors,
 	black: '#40474e',
 	default: '#1fdc6b',
 	tomato: 'FF5238',
@@ -28,14 +40,20 @@ const colors = {
 	background: '#F7FAFC',
 	altBackground: '#fafffd'
 };
-const breakpoints = [ '40em', '52em', '64em' ];
 
-const theme = {
-	...chakraTheme,
+const breakpoints = createBreakpoints({
+	sm: '40em',
+	md: '52em',
+	lg: '64em',
+	xl: '80em',
+})
+
+
+const theme = extendTheme({
 	colors,
 	fonts,
 	fontSizes,
 	breakpoints,
-};
+})
 
 export default theme;

--- a/package.json
+++ b/package.json
@@ -25,12 +25,12 @@
 		"deploy": "gh-pages -d example/build"
 	},
 	"peerDependencies": {
-		"@chakra-ui/core": "^1.0.0-rc.8",
+		"@chakra-ui/react": "^1.0.0",
 		"framer-motion": "^2.9.4",
 		"react": "^16.0.0"
 	},
 	"devDependencies": {
-		"@chakra-ui/core": "^1.0.0-rc.8",
+		"@chakra-ui/react": "^1.0.1",
 		"@emotion/react": "^11.1.1",
 		"@emotion/styled": "^11.0.0",
 		"@testing-library/jest-dom": "^4.2.4",
@@ -65,6 +65,9 @@
 		"rimraf": "^3.0.2",
 		"tsdx": "^0.14.1",
 		"tslib": "^2.0.3",
+		"downshift": "^6.0.6",
+		"react-highlight-words": "^0.16.0",
+		"react-use": "^15.3.4",
 		"typescript": "^3.7.5"
 	},
 	"files": [
@@ -87,8 +90,5 @@
 		"url": "git+https://github.com/koolamusic/chakra-ui-autocomplete.git"
 	},
 	"dependencies": {
-		"downshift": "^6.0.6",
-		"react-highlight-words": "^0.16.0",
-		"react-use": "^15.3.4"
 	}
 }

--- a/src/index.test.tsx
+++ b/src/index.test.tsx
@@ -2,7 +2,7 @@ import { CUIAutoComplete } from './index'
 
 import React from 'react'
 import renderer from 'react-test-renderer'
-import { Text, Flex, Heading, Avatar } from '@chakra-ui/core'
+import { Text, Flex, Heading, Avatar } from '@chakra-ui/react'
 
 export interface Item {
   label: string

--- a/src/index.tsx
+++ b/src/index.tsx
@@ -11,7 +11,7 @@ import useDeepCompareEffect from 'react-use/lib/useDeepCompareEffect'
 import { FormLabel, FormLabelProps } from '@chakra-ui/form-control'
 import { Text, Stack, Box, BoxProps, List, ListItem, ListIcon } from '@chakra-ui/layout'
 import { Button, ButtonProps } from '@chakra-ui/button'
-import { Input, InputProps } from '@chakra-ui/core'
+import { Input, InputProps } from '@chakra-ui/input'
 import { IconProps } from '@chakra-ui/icon'
 import { Tag, TagCloseButton, TagLabel, TagProps } from '@chakra-ui/tag'
 

